.github/workflows/update-3rdparty.yml

```diff
@@ -0,0 +1,254 @@
+name: Update 3rdparty sources
+on:
+  push:
+#     branches:
+#       - 'release-**'
+
+  schedule:
+    - cron:  '0 0 * * fri'
+
+jobs:
+  update-iremapper-source:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          ref: development
+        
+      - name: Download latest IRE mapper
+        uses: carlosperate/download-file-action@v1.0.0
+        with:
+          file-url: https://raw.githubusercontent.com/IRE-Mudlet-Mapping/ire-mapping-script/gh-pages/downloads/mudlet-mapper.xml
+          file-name: mudlet-mapper.xml
+          location: src/
+
+      # strictly speaking this step isn't necessary since the PR action can filter as well, but it's easier
+      # to see in github UI's if PR creation was skipped when we have an explicit pre-check
+      - name: check if we have any updates
+        id: getdiff
+        run: |
+          lines_changed=$(git diff --patch --unified=0 src/mudlet-mapper.xml | wc --lines)
+          echo "$lines_changed lines changed."
+          echo ::set-output name=lines_changed::$lines_changed
+          
+      - name: send in a pull request
+        uses: gr2m/create-or-update-pull-request-action@v1.x
+        if: steps.getdiff.outputs.lines_changed > 0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          title: "Update bundled IRE mapper script"
+          body: |
+            #### Brief overview of PR changes/additions
+            :crown: An automated PR to update the IRE mapper script to the latest version.
+            #### Motivation for adding to Mudlet
+            So people don't get attacked with a "your mapping script is out of date!" notification as soon as they create a profile for a new IRE game.
+            #### Other info (issues closed, discussion etc)
+            _update triggered by ${{ github.ref }} ${{ github.sha }}_
+          branch: update-ire-mapping-script-${{ github.sha }}
+          path: src/
+          commit-message: (autocommit) Updated IRE mapping script to latest upstream
+          author: "mudlet-machine-account <mudlet-machine-account@users.noreply.github.com>"
+
+  update-edbee-fork:
```

Comment:
Job title mentions source, so we can get an [easy overview](https://github.com/Mudlet/Mudlet/actions/runs/57384020) of where the update happened from - and be mindful that in case of a fork, the source repository might have an update that this won't show.